src/App.tsx

```diff
@@ -1,15 +1,31 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const handleQueryChange
+    = (event: {
+      target:
+      { value: React.SetStateAction<string>; };
+    }) => {
+      setQuery(event.target.value);
+    };
+
+  const searchWord = query.toLowerCase();
+  const visibleMovies = moviesFromServer
+    .filter(
+      (movie) => movie.title.toLowerCase().includes(searchWord)
```

Comment:
if some code written twice it can be placed in one function.
Create function for your `title.toLowerCase().includes(searchWord)`

Reply:
thanks for the feedback!